### PR TITLE
Changed config file based on newer config parameters

### PR DESCRIPTION
**Pull request by udaysk23**
Changed config file based on newer config parameters

Signed-off-by: udaysk23 <ukurundw@redhat.com>

### CODE REVIEW

File: rgw/COSBench/haproxy.cfg [hunk 1]
```diff
@@ -64,7 +64,8 @@ defaults
 
 
 # JTH start
-listen  stats   0.0.0.0:1936
+listen  stats
```
Comment (rakeshgm) [inline rgw/COSBench/haproxy.cfg:67]: looks like a format issue

File: rgw/COSBench/haproxy.cfg [hunk 2]
```diff
@@ -93,7 +94,8 @@ listen  stats   0.0.0.0:1936
 #---------------------------------------------------------------------
 # main frontend which proxys to the backends
 #---------------------------------------------------------------------
-frontend  main *:5000
+frontend  main
+    bind    *:5000
```
Comment (rakeshgm) [inline rgw/COSBench/haproxy.cfg:98]: same here, looks like a format issue

File: rgw/COSBench/haproxy.cfg [hunk 3]
```diff
@@ -64,7 +64,8 @@ defaults
 
 
 # JTH start
-listen  stats   0.0.0.0:1936
+listen  stats
+        bind    0.0.0.0:1936
```
Comment (rakeshgm) [inline rgw/COSBench/haproxy.cfg:68]: looks like a formatting issue here.

File: rgw/COSBench/haproxy.cfg (hunk 2 above)
Comment (udaysk23) [reply rgw/COSBench/haproxy.cfg:98]: I have verified these parameters on my environment and these parameters are working in my environment

**Review by rakeshgm — APPROVED** [review]

**Pull request merged**